torch/_inductor/memory.py

```diff
@@ -545,11 +549,22 @@ class PeakMemoryResult:
             peak_memory, _ = estimate_peak_memory(
                 order, name_to_input_buf, graph_outputs
             )
-            peak_memory_diff_methods.append(PeakMemoryResult(order, peak_memory))
-            torch_log.warning("%s peak memory: %d", method.__name__, peak_memory)
+            peak_memory_diff_methods.append(
+                PeakMemoryResult(order, peak_memory, method.__name__)
+            )
+            torch_log.info("%s peak memory: %d", method.__name__, peak_memory)
         except Exception as e:
             torch_log.error("Failed to reorder for %s: %s", method.__name__, e)
 
+    # logging to scuba table (fb internal)
```

Comment:
maybe we can remove this comment (OSS folks shouldn't need to know about scuba)

Reply:
removed.